semgrep_output_v1.atd

```diff
@@ -1104,6 +1104,8 @@ type found_dependency = {
     <ts repr="map">;
   ?resolved_url: string option;
   transitivity: transitivity;
+  (* lockfile_path and line_number point to some location in some file *)
```

Comment:
this comment is not very useful. I'm sure they point to "some" file, would be good to specifiy more precisely what file it is.
Maybe an example at the beginniing of this type would help to understand the type.